src/style.css

```diff
@@ -1 +1,49 @@
 /* add styles here */
+* {
+  margin: 0;
+  padding: 0;
+  border: 0;
+  font-size: 100%;
+}
```

Comment:
You are using the `*` selector to apply styles to all elements. This is not recommended as it impacts the performance of your CSS. Refer to the checklist item: 'Don't use * selector (it impacts performance)'. Try to apply these styles only to the elements that require them.